scripts/number_clusters.py

```diff
@@ -16,11 +16,14 @@
 
     # Read each fastbaps file
     for fb_file in snakemake.input:
+        print(snakemake.input)
+        print(fb_file)
```

Comment:
I think these print statements should be removed?

Reply:
yes